package.json

```diff
@@ -30,11 +30,15 @@
     "lint": "npm run lint:code",
     "lint:code": "eslint src test",
     "lint:docs": "documentation lint src/index.js",
-    "prepare": "npm test && npm run build",
+    "publish:dev": "npm run test:build && sh ci/publish-version.sh prerelease --preid=alpha",
```

Comment:
I don't if we need prerelease capability

Reply:
Yeah, maybe it's not necessary.